Sources/PaymentButtons/ImageAsset.swift

```diff
@@ -9,24 +9,15 @@ enum ImageAsset {
 
         case .payLater:
             imageAssetString = "paypal"
-
-            if button.size == .collapsed {
-                imageAssetString = "paypal_monogram"
-            }
-
+            
         case .credit:
-            if button.size == .collapsed {
-                imageAssetString = "credit_monogram"
-            } else if button.size != .miniWithWordmark && button.size != .mini {
+            if button.size != .mini {
                 imageAssetString = "credit"
             }
         }
-
-        if button.size == .mini {
-            imageAssetString = "paypal_monogram"
-        } else if button.size == .miniWithWordmark {
-            imageAssetString = "paypal_vertical"
-        }
+            if button.size == .mini {
+                imageAssetString = "paypal_vertical"
+            }
```

Comment:
This spacing looks off, can we cmd + i ?

Reply:
I checked the spacing and also used cmd + i; Looks good